src/DIRAC/WorkloadManagementSystem/Agent/PilotLoggingAgent.py

```diff
@@ -0,0 +1,103 @@
+""" :mod: PilotLoggingAgent
+
+    PilotLoggingAgent sends Pilot log files to an SE
+"""
+
+# # imports
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
These `__future__` imports can be removed (also below).

Reply:
Gone.